tests/test_trash.py

```diff
@@ -15,8 +16,11 @@ def test_version(self, portals, dbus_con):
 
     def test_trash_file_fails(self, portals, dbus_con):
         trash_intf = xdp.get_portal_iface(dbus_con, "Trash")
-        with open("/proc/cmdline") as fd:
-            result = trash_intf.TrashFile(fd.fileno())
+        try:
+            with open("/proc/cmdline") as fd:
+                result = trash_intf.TrashFile(fd.fileno())
+        except PermissionError as e:
+            pytest.skip(f"Couldn't open file: {e}")
```

Comment:
Should be a bit more specific about what it's trying to open, or even better, find something that works everywhere.